app/src/main/java/me/banes/chris/tivi/inject/AppModule.kt

```diff
@@ -45,7 +48,12 @@ class AppModule {
     @Singleton
     @Provides
     fun provideRxSchedulers(): AppRxSchedulers {
-        return AppRxSchedulers()
+        return AppRxSchedulers(
+                Schedulers.single(),
+                Schedulers.io(),
+                Schedulers.io(),
```

Comment:
are these duplicate io() params intended?

Reply:
Yep, they're used for different things.